test/question.spec.js

```diff
@@ -5,13 +5,14 @@ import { expect } from 'chai';
 
 import server from '../index';
 
-describe('Question Api Exists', () => {
+describe('Questions', () => {
   describe('GET /questions', () => {
     it('should return status code 200 on successful questions fetching', (done) => {
       request(server)
         .get('/api/v1/questions')
         .end((err, res) => {
           expect(res.statusCode).to.equal(200);
+          expect(res.body).to.be.an('object');
```

Comment:
Nice job here on trying to check what the response body is but you just checking that it is an object what about checking specific values in the response body.

Reply:
Alright, thank you. I will make the necessary changes